test.ps1

```diff
@@ -2,10 +2,10 @@
 #requires -version 4
 [CmdletBinding(PositionalBinding = $true)]
 param(
-    [Parameter]
-    [string]$Command = 'default-build',
     [Parameter(Mandatory = $true)]
     [string]$RepoPath,
+    [Parameter(Position = 1)]
```

Comment:
Position = 0?

Reply:
On second thought, this doesn't apply in 2.0.0. This was before the run.ps1 factoring, so we can drop this parameter altogether